planning/behavior_path_planner/src/path_utilities.cpp

```diff
@@ -195,13 +195,13 @@ std::pair<TurnIndicatorsCommand, double> getPathTurnSignal(
   if (path.shift_length.empty()) {
     return std::make_pair(turn_signal, max_distance);
   }
-  const auto base_link2front = common_parameter.base_link2front;
-  const auto vehicle_width = common_parameter.vehicle_width;
+  const auto & base_link2front = common_parameter.base_link2front;
+  const auto & vehicle_width = common_parameter.vehicle_width;
   const auto shift_to_outside = vehicle_width / 2;
-  const auto tl_on_threshold_lat = common_parameter.turn_light_on_threshold_dis_lat;
-  const auto tl_on_threshold_long = common_parameter.turn_light_on_threshold_dis_long;
-  const auto prev_sec = common_parameter.turn_light_on_threshold_time;
-  const double epsilon = 1e-6;
+  const auto & tl_on_threshold_lat = common_parameter.turn_light_on_threshold_dis_lat;
+  const auto & tl_on_threshold_long = common_parameter.turn_light_on_threshold_dis_long;
+  const auto & prev_sec = common_parameter.turn_light_on_threshold_time;
+  constexpr double epsilon = 1e-6;
```

Comment:
In my understanding, we don't have to add reference to primitive types (int, double, ...)

Reply:
Is this what clang-tidy recommended?

Reply:
Yes, that is true. We dont have to pass simple primitive by reference. Clang tidy also didn't recommend anything.

The reason I added reference is that I thought the original intention is to "alias" the parameters, so it might as well be const reference. 

In case there is concern about them, I can actually removed the reference.

Reply:
After some consultation, indeed the normal practices is to copy the primitive (or simple structs). Therefore I removed the reference in the new update.

Reply:
Thank you!
It would be better to add the commit link for the changes to resolve this conversation.

If I were you, I would comment like this

DONE
https://github.com/autowarefoundation/autoware.universe/pull/1832/commits/022b5c2811ca7f722630a2bdc4a0c9d5789ef67b